core/src/main/resources/swagger/rawls.yaml

```diff
@@ -2566,7 +2632,10 @@ paths:
           description: Rawls Internal Error
           schema:
             $ref: '#/definitions/ErrorReport'
-      description: Rename method configuration
+      description: |
+        Rename method configuration.
+        The workspace name and namespace in the HTTP body are ignored.
```

Comment:
you can't rename across workspaces. "why are they there at all then?" excellent question. i looked into fixing this but it spidered out into breaking changes for practically every method config endpoint.